src/Pages/ChatRoom/ChatRoom.tsx

```diff
@@ -0,0 +1,59 @@
+import HomeBar from "HomeBar";
+import StatusBar from "StatusBar";
+import BottomInputBox from "./BottomInputBox";
+import TopInChat from "./TopInChat";
+import { useState } from "react";
+import InchatList from "./InchatList";
+import user from "user.json";
+import { styled } from "styled-components";
+
+const ChatRoom: React.FC = () => {
+  const [chat, setChat] = useState<
+    Array<{ value: string; id: number; sender: string; date: string }>
+  >([]);
+  const [currentUser, setCurrentUser] = useState(user.users[0]);
```

Comment:
```suggestion
type ChatItem = {
  value: string;
  id: number;
  sender: string;
  date: string;
};

type User = {
  id: number;
  name: string;
};

const ChatRoom: React.FC = () => {
  const [chat, setChat] = useState<ChatItem[]>([]);
  const [currentUser, setCurrentUser] = useState<User>(user.users[0]);

```
type은 따로 빼면 한눈에 보기 편할 것 같아요!